### PR TITLE
Add Panel 1.3 release blog post

### PR DESCRIPTION
**Pull request by philippjfr**

### CODE REVIEW

File: content/panel_release_1.3.ipynb [hunk 1]
```diff
@@ -0,0 +1,1970 @@
+{
```
Comment (ahuang11) [inline content/panel_release_1.3.ipynb:1]: <p>Should <code>PanelCallbackHandler</code> be linked to <a href="https://panel.holoviz.org/reference/chat/PanelCallbackHandler.html" target="_blank">https://panel.holoviz.org/reference/chat/PanelCallbackHandler.html</a></p><p> </p><p>And maybe more specific instructions about <code>call us</code> (inviting them to our Discord / Discourse). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/holoviz-dev/blog/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (ahuang11) [inline content/panel_release_1.3.ipynb:1]: <p>Didn't realize there was in line commenting:</p><p><a href="https://app.reviewnb.com/holoviz-dev/blog/pull/26/discussion" target="_blank">https://app.reviewnb.com/holoviz-dev/blog/pull/26/discussion/</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/holoviz-dev/blog/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (ahuang11) [inline content/panel_release_1.3.ipynb:1]: <p>Can we add an interactive demo of <code>TextAreaInput(placeholder="Enter new lines to see this grow!", auto_grow=True, max_rows=4)</code> too? I think it's quite relevant to ChatInterface.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/holoviz-dev/blog/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

**Comment by maximlt** [comment]
Integrated in https://github.com/holoviz-dev/blog/pull/27

**Comment by maximlt** [comment]
@ahuang11 do you still want to make changes to this blog post? If not, let's close this PR :)

**Comment by ahuang11** [comment]
Sure feel free to close.

**Pull request closed without merging**